app/views/dashboard/works/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<%= content_for :navbar_items do %>
+  <li class="nav-item">
+    <%= link_to(t('.back'),
+                resource_path(@work.uuid),
+                class: 'btn btn-outline-light btn--squish mr-lg-2 my-1') %>
+  </li>
+<% end %>
+
+<%= content_for :body_attributes, 'data-spy="scroll" data-target="#navbar-settings" data-offset="10" style="position: relative"'.html_safe %>
+
+<div class="container-fluid">
+  <article class="row">
+    <div class="col-lg-3">
+      <aside class="sidebar sticky-top">
+        <ul id="navbar-settings" class="nav flex-column">
+          <li class="nav-item">
+            <%= link_to t('.visibility.heading'), "##{t('.visibility.heading').parameterize}", class: 'nav-link' %>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#<%= t('.doi.heading').parameterize %>"><%= t('.doi.heading') %></a>
+          </li>
+        </ul>
+      </aside>
+    </div>
+    <div class="col-lg-9">
+
+        <h1 class="h2 mb-3"><%= t('.heading', work_title: @work.decorated_latest_published_version.title) %></h1>
+
+        <div class="keyline keyline--left">
+          <h2 id="<%= t('.visibility.heading').parameterize %>" class="h4"><%= t('.visibility.heading') %></h2>
+        </div>
+
+        <p><%= t('.visibility.explanation') %></p>
+
+        <%- visibility_form_id = "edit_visibility_#{dom_id @work}" %>
+        <%= form_for [:dashboard, @work], remote: false, html: { id: visibility_form_id, class: 'edit-work-visibility' } do |visibility_form| %>
+          <%= hidden_field_tag 'form_id', visibility_form_id %>
+          <% if params['form_id'] == visibility_form_id %>
+            <%= render FormErrorMessageComponent.new(form: visibility_form) %>
+          <% end %>
```

Comment:
☝️ This is a thing I'm playing around with. We are going to have _multiple forms_ on this page. Many of them will all submit to the same endpoint, `Dash::WorksController#update`. So if there is an error in the form submission and this page is re-rendered, how do you know which form to show the errors on? They all use `@work`, they would _all_ simultaneously show the same error message, even if it doesn't affect them.

One idea I had was each form could have a hidden field called `form_id`, with a unique value for each form. If the form was submitted, had validation errors, and this page got-rendered, each form would "know" whether or not they had made the original submission, and therefore if they should show the errors or not. It's a work in progress, but it does seem to be the simplest and most foolproof solution I've thought of.